apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js

```diff
@@ -222,11 +223,25 @@ const CreatePaymentByLinkService = new GQLCustomSchema('CreatePaymentByLinkServi
                     multiPaymentId = id
                 }
 
+                const multiPayment = await MultiPayment.getOne(context, { id: multiPaymentId })
+
                 return {
                     multiPaymentId,
+                    amount: multiPayment.amountWithoutExplicitFee,
+                    serviceCharge: multiPayment.explicitServiceCharge, // explicit* are paid by resident
+                    explicitFee: multiPayment.explicitFee,
+                    totalAmount: multiPayment.amount,
                     address: normalizedAddress.address,
-                    unitName: normalizedAddress.addressMeta.unitName,
+                    addressMeta: {
+                        dv: 1,
+                        value: get(normalizedAddress, ['addressMeta', 'value'], ''),
+                        unrestricted_value: get(normalizedAddress, ['addressMeta', 'unrestricted_value'], ''),
+                        data: get(normalizedAddress, ['addressMeta', 'data'], null),
+                    },
+                    unitType: get(normalizedAddress, ['addressMeta', 'unitType']),
```

Comment:
Maybe set `flat` unitType by default? 
```suggestion
                    unitType: get(normalizedAddress, ['addressMeta', 'unitType', 'flat']),
```

Reply:
Looks like it does not matter here